pkg/scheduler/framework/cycle_state_test.go

```diff
@@ -41,6 +41,7 @@ func TestCycleStateClone(t *testing.T) {
 		data: data1,
 	}
 	state.Write(key, originalValue)
+	state.SetRecordPluginMetrics(true)
```

Comment:
can we test both true and false?

Reply:
👍 
Changed the whole test to be table-driven test and added the test for that as a test case.